package.json

```diff
@@ -12,8 +12,9 @@
     "postinstall": "npm prune && npm run build",
     "prepush": "opt --out prepush --exec 'npm run validate'",
     "start": "webpack-dev-server --config webpack/starterApp.js --devtool eval --progress --colors --hot --history-api-fallback --open",
-    "test": "NODE_ENV=test karma start",
-    "test:watch": "npm test -- --auto-watch --no-single-run --reporters progress",
+    "test": "jest",
+    "test:watch": "npm test -- --verbose --notify --watch",
```

Comment:
Are there options for different reporters or such?  I like verbose output when I do `npm test`, but for `test:watch`, I want something nice and terse so it fits in a small terminal inside my editor.  With Karma, we were defaulting to verbose output and then overriding it for `test:watch`.